core/src/main/java/org/lflang/generator/LFGenerator.java

```diff
@@ -109,6 +117,12 @@ public void doGenerate(Resource resource, IFileSystemAccess2 fsa, IGeneratorCont
 
     } else {
 
+      // If "-c" or "--clean" is specified, delete any existing generated directories.
+      cleanIfNeeded(lfContext);
+
+      // If @property annotations are used, run the LF verifier.
+      runVerifierIfPropertiesDetected(resource, lfContext);
```

Comment:
Is there a reason for calling the verifier here instead of in GeneratorBase?

I spend some time today wondering why the verifier tests pass on my machine, although I don't have uclid installed. Turns out that the errors reported by the verifier are ignored, because the message reporter is reset at the beginning of `doGenerate()` in GeneratorBase. So it seems like the verifier should be called in GeneratorBase, after cleaning and resetting the message reporter.

Reply:
At the time when I implemented this,  it felt a bit more natural to call the verifier outside of GeneratorBase, because it focuses on generating target source code.

But I am certainly not against moving it into `GeneratorBase`. :-)